repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,147 @@
+{
+  "type": "object",
+  "properties": {
+    "marathon/libprocess-ip": {
```

Comment:
My idea here is that there are a bunch of docs related to the LIBPROCESS_IP environment variable, so I think it's nice to make it explicit that these are the same thing.